documentation/modules/ROOT/pages/configuration/storage.adoc

```diff
@@ -0,0 +1,635 @@
+
+|===
+
+== Memory
+
+`MemoryOffsetBackingStore` is a volatile, in-memory store used by Debezium Embedded to track source offsets. It keeps the offset state only during the application's runtime and loses it on shutdown or crash. It's ideal for testing or short-lived tasks but not suitable for production where offset persistence is needed.
+
+=== Offset Store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[offset-storage-memory]]<<offset-storage-memory,`offset.storage`>>
+|No default
+|Must be set to `org.apache.kafka.connect.storage.MemoryOffsetBackingStore`
+
+|===
+
+=== Internal schema history store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[schema-history-internal-memory]]<<schema-history-internal-memory,`schema.history.internal`>>
+|No default
+|Must be set to `io.debezium.relational.history.MemorySchemaHistory`
+
+|===
+
+
+== JDBC
+The store uses arbitrary relational database to store the data for which an appropriate link:https://en.wikipedia.org/wiki/JDBC_driver[JDBC driver] is provided.
+Either the source database can be used as the storage or any other database.
+
+The users either use the default pre-configured DML and DDL statements or can override them with their own either for database dialect compatibility or to tailor them to their use case.
+
+
+=== Offset Store
+
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[jdbc-offset-type]]<<jdbc-offset-type, `offset.storage`>>
+|No default
+|Must be set to `io.debezium.storage.jdbc.offset.JdbcOffsetBackingStore`.
+
+|[[jdbc-offset-url]]<<jdbc-offset-url, `offset.storage.jdbc.connection.url`>>
+|No default
+|JDBC driver connection string to connect to the database.
+
+|[[jdbc-offset-user]]<<jdbc-offset-user, `offset.storage.jdbc.connection.user`>>
+|No default
+|(Optional) Username of the database which will be used to access the database storage.
+
+|[[jdbc-offset-password]]<<jdbc-offset-password, `offset.storage.jdbc.connection.password`>>
+|No default
+|(Optional) Password of the database which will be used to access the database storage.
+
+|[[wait-retry-delay-ms]]<<wait-retry-delay-ms, `offset.storage.jdbc.connection.wait.retry.delay.ms`>>
+|3 seconds
+|(Optional) Delay of retry on wait for connection failure.
+
+|[[retry-max-attempts]]<<retry-max-attempts, `offset.storage.jdbc.connection.retry.max.attempts`>>
+|5
+|(Optional) Maximum number of retry attempts before giving up.
+
+|[[jdbc-offset-table-name]]<<jdbc-offset-table-name, `offset.storage.jdbc.table.name`>>
+|`debezium_offset_storage`
+|The name of the table to which the offsets will be stored.
+
+|[[jdbc-offset-table-ddl]]<<jdbc-offset-table-ddl, `offset.storage.jdbc.table.ddl`>>
+|link:https://raw.githubusercontent.com/debezium/debezium/main/debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/offset/JdbcOffsetBackingStoreConfig.java[source code]
```

Comment:
I'd prefer a separate section under the options table that would should DDLs instead of links to the source code